com.polydez.KeyVault.yml

```diff
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.polydez.KeyVault
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: keyvault
+separate-locales: false
+finish-args:
+  # Basic permissions, adjust if your app needs different access
+  - --socket=fallback-x11 # Needed for X11 display server
+  - --socket=wayland      # Needed for Wayland display server
+  - --share=network     # Uncomment if the app needs internet access
+  - --share=ipc           # Allow inter-process communication
+  - --device=dri          # For graphics hardware acceleration
+
+
+modules:
+  - name: keyvault
+    buildsystem: simple
+    build-commands:
+      # Extract the downloaded archive first
+      - unzip keyvault-linux.zip
+
+      # Create the bundled application directory and standard /app/bin
+      - install -d /app/keyvault
+      - install -d /app/keyvault/lib
+      - install -d /app/keyvault/data
+      - install -d /app/bin
+
+      # Install the main executable binary into the bundle directory
+      - install -Dm755 keyvault /app/keyvault/
+
+      # List current directory contents for debugging
+      - ls -la
+
+      # Install the necessary shared libraries into the bundle's lib directory
+      - cp -r lib/* /app/keyvault/lib/
+
+      # Install the necessary Flutter data files into the bundle's data directory
+      - cp -r data/* /app/keyvault/data/
+
+      # Create a symlink in /app/bin pointing to the bundled executable
+      - ln -s /app/keyvault/keyvault /app/bin/keyvault
+
+      # Install the application icon (extracted from the source archive)
+      # Update path for 512x512 dimension
+      - install -Dm644 keyvault.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      # Add other icon sizes if present
+
+      # Install the pre-created .desktop file (copied from local file)
+      - install -Dm644 com.polydez.KeyVault.desktop /app/share/applications/${FLATPAK_ID}.desktop
+
+      # Install the pre-created .metainfo.xml file (copied from local file)
+      - install -Dm644 com.polydez.KeyVault.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      
+    sources:
+      # Main application archive
+      - type: file
+        dest-filename: keyvault-linux.zip
+        url: http://polydez.com/download/keyvault-linux.zip
```

Comment:
the URL should be versionned. Otherwise it will randomly break.

Reply:
I'll update that now :)